src/Applications/LDAS_App/process_rst.py

```diff
@@ -0,0 +1,79 @@
+import os
+import sys
+import subprocess
+import shutil
+import glob
+import ruamel.yaml
+import shlex
+from remap_utils import *
+
+def remap_config_ldas(config, RESTART_str, RESTART_PATH, RESTART_ID):
+   yyyymmddhh = config['input']['shared']['yyyymmddhh']
+   MODEL      = config['input']['surface']['catch_model']
+   out_dir    = config['output']['shared']['out_dir']
+   # MERRA2
+   if RESTART_str == "M" :
+     config['input']['shared'] = merra2_expid(config['input']['shared'])
+     config['input']['shared']['rst_dir'] = out_dir+ '/merra2_tmp_'+ yyyymmddhh
+     config['input']['surface']['wemin'] = 26
+     config['input']['shared']['bcs_dir'] = '/gpfsm/dnb02/ltakacs/bcs/Ganymed-4_0/Ganymed-4_0_MERRA-2/CF0180x6C_DE1440xPE0720/'
+     
+   if RESTART_str == "G" :
+     # WY note: it is a bad idea to overload restart_path and restart_id
+     config['input']['surface']['catch_tilefile'] = os.path.realpath(RESTART_ID+'scratch/tile.data')
+     config['input']['shared']['rst_dir']  = os.path.dirname(RESTART_PATH) + '/'
+     config['input']['surface']['wemin'] = 13
+     if 'NLv' in config['input']['surface']['catch_tilefile'] : config['input']['surface']['wemin'] = 26
+     
+   if RESTART_str == "F" :
+     date_16 = 20170124 
+     date_17 = 20171101 
+     date_21 = 20180711 
+     date_22 = 20190313
+     date_25 = 20200130
```

Comment:
This history of FP systems is a bit outdated.  GEOS 5.27 went operational on ~25 Feb 2021 and was replaced with GEOS 5.29 on ~1 Mar 2022 (https://gmao.gsfc.nasa.gov/GMAO_products/NRT_products.php).  We should add these two new versions here, otherwise I expect ldas_setup to fail when a restart date after 25 Feb 2021 is requested.

Reply:
Good point.  Having this with remap_restart.py (i.e., in GMAO_Shared) probably makes more sense.

Reply:
I updated here. But will move to remap_restart later.